js/modal.js

```diff
@@ -0,0 +1,137 @@
+    details:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    featured_image: './assets/images/screenshot4.png',
+    lang_list: ['javascript', 'html', 'css'],
+    breadcrumbs: ['Home', 'Works', 'Calenda'],
+    source: 'https://github.com/niicode/portfolio',
+    live_demo: 'https://niicode.github.io/portfolio/',
+  },
+  {
+    title: 'GymFit',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    details:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    featured_image: './assets/images/screenshot1.png',
+    lang_list: ['javascript', 'html', 'css'],
+    breadcrumbs: ['Home', 'Works', 'Calenda'],
+    source: 'https://github.com/niicode/portfolio',
+    live_demo: 'https://niicode.github.io/portfolio/',
+  },
+];
+
+const workCardShouldReverse = (index) => {
+  if (index % 2 === 0) {
+    return 'row-reverse work-item-2';
+  }
+  return 'work-item-1';
+};
+
+const breadcrumbsHtml = (workObj, key, bread = '') => {
+  /* eslint-disable no-restricted-syntax */
+  for (const breadcrumb in workObj[key].breadcrumbs) {
+    if (breadcrumb) {
+      bread += `<li>${workObj[key].breadcrumbs[breadcrumb]}</li>`;
```

Comment:
- [ ] Please make sure that this list of bread crumbs looks similar to that of the Figma template.
### Your implementation
![image](https://user-images.githubusercontent.com/45460192/179794479-7c37f431-b4dd-4df5-a0b3-f2b4c1703d94.png)
### Figma
![image](https://user-images.githubusercontent.com/45460192/179794574-c08e9f0e-6349-438f-a734-4a18501f1d60.png)